### ISSUE TITLE
Fix S3010 FN: Support unsigned right-shift operator (>>>)

### ISSUE THREAD

**Issue by gregory-paidis-sonarsource** [SonarSource/sonar-dotnet#6303]
### Description

FN for S3010 (StaticFieldWrittenFromInstanceConstructor).

Using C#11 unsigned right-shift operator (>>>), the rule is not raising an issue.

### Repro steps

```csharp
public class Person
{
    private static int expectedFingers; // FN

    public Person(DateTime birthday)
    {
        expectedFingers >>>= 5; // FN
    }
}
```

### Expected behavior

The rule should raise an issue.

### Actual behavior

The rule does not raise an issue.
